scene/change-atmosphere-effect/src/main/java/com/esri/samples/change_atmosphere_effect/ChangeAtmosphereEffectSample.java

```diff
@@ -80,39 +77,66 @@ public void start(Stage stage) {
       Camera camera = new Camera(64.416919, -14.483728, 100, 318, 105, 0);
```

Comment:
might as well `var` this one up! 
```suggestion
      var camera = new Camera(64.416919, -14.483728, 100, 318, 105, 0);
```